src/Common/TsVectorSubscriber.php

```diff
@@ -73,9 +74,40 @@ public function loadClassMetadata(LoadClassMetadataEventArgs $eventArgs)
     {
         /** @var ClassMetadata $metaData */
         $metaData = $eventArgs->getClassMetadata();
-
         $class = $metaData->getReflectionClass();
+
         foreach ($class->getProperties() as $prop) {
+            // check for php8 attributes on the class properties
+            if (method_exists($prop, 'getAttributes')) {
+                foreach ($prop->getAttributes() as $reflectionAttribute) {
+                    if ($reflectionAttribute->getName() === TsVector::class) {
+//                    /** @var TsVector $attribute */
+                    $attribute = new ($reflectionAttribute->getName())(...$reflectionAttribute->getArguments()); // crazy, something is wrong here.
```

Comment:
I'm a little apprehensive about this comment!

Reply:
No doubt!  I'll see if I can figure it out.  I'm fighting a little bit with supporting both attributes and annotations, I think that attributes should take priority if both exist.